src/services/counters/CounterService.ts

```diff
@@ -102,12 +114,19 @@ export class CounterService {
   }
 
   reset(container: ICacheContainer, counter: ICounter): ng.IPromise<any> {
-    return this.jGroupsService.getServerGroupCoordinator(container.serverGroup).then((coord: IServerAddress) => {
+    if (this.launchType.isStandaloneLocalMode()) {
       return this.dmrService.executePost({
-        address: this.getCounterRuntimeAddress(counter, container.name, coord.host, coord.name),
+        address: this.getCounterRuntimeAddress(counter, container.name),
         operation: "counter-reset"
       });
-    });
+    } else {
```

Comment:
You don't handle `isStandaloneClusteredMode` here which I think is why it fails with standalone.sh -c clustered.xml

Reply:
I'll check - it works for me. Don't forget that you also need to remove counters=COUNTERS on fresh restart of the test between standalone and clustered

Reply:
Why? If you are starting the server with different configurations then the state is persisted to the currently loaded config. So starting the server with `standalone.sh` and `standalone.sh -c clustered.xml` should be separate.